app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -207,6 +210,25 @@ class AwesomeBarView(
         searchSuggestionProviderMap = HashMap()
     }
 
+    private fun getSearchEngineSuggestionsHeader(): String? {
+        val searchState = activity.components.core.store.state.search
+        var searchEngine = searchState.selectedOrDefaultSearchEngine?.name
```

Comment:
nit - maybe injecting components as param to this method and using one variable would be enough as well :)